lib/go/thrift/header_protocol.go

```diff
@@ -345,20 +345,6 @@ func (p *THeaderProtocol) SetTConfiguration(cfg *TConfiguration) {
 	p.cfg = cfg
 }
 
-// GetResponseHeadersFromClient is a helper function to get the read THeaderMap
-// from the last response received from the given client.
-//
-// If the last response was not sent over THeader protocol,
-// a nil map will be returned.
-func GetResponseHeadersFromClient(c TClient) THeaderMap {
```

Comment:
This function was in a tagged release. Maybe mark it as deprecated instead of removing?

Reply:
It was added after 0.13.0 so I don't believe it's in any tagged release :)

Reply:
At least it's not here: https://pkg.go.dev/github.com/apache/thrift@v0.13.0/lib/go/thrift :)

Reply:
Ah, I was confused because the commit date was earlier than the 0.13 tag :)